src/test/java/uk/gov/ons/ctp/response/action/client/CaseSvcClientServiceTest.java

```diff
@@ -44,17 +45,19 @@ public void setup() {
   }
 
   @Test
-  public void ensureNewIacGenerated() throws Exception {
-
-    UriComponents uriComponents = mock(UriComponents.class);
+  public void ensureNewIacGenerated() {
+    CaseSvc caseSvc = new CaseSvc();
+    UriComponents uriComponents =
```

Comment:
It feels a bit weird making a real CaseSvc and then mocking it out below, but I get why

Reply:
Yeah agree,  in the future that'll hopefully get looked at.